src/cli.ts

```diff
@@ -1,32 +1,94 @@
-import chalk from 'chalk'
-import parseArgs from 'yargs-parser'
+import chalk from "chalk";
+import parseArgs from "yargs-parser";
+import path from "path";
 
-import prepareApk from '.'
+import { prepareApk, prepareAppBundle } from ".";
 
 async function main() {
   const args = parseArgs(process.argv.slice(2), {
-    string: ['apktool'],
-  })
+    string: ["apktool"]
+  });
 
   if (args.help) {
-    showHelp()
-    process.exit()
+    showHelp();
+    process.exit();
   }
 
-  const [apkPath] = args._
-  if (!apkPath) {
-    showHelp()
-    process.exit(1)
+  const [filePath] = args._;
+  if (!filePath) {
+    showHelp();
+    process.exit(1);
   }
+  const fileExtension: string = path.extname(filePath);
+  const finishedFileName = `${path.basename(
+    filePath,
+    fileExtension
+  )}-patched.${fileExtension}`;
+  switch (fileExtension) {
+    case "apk":
```

Comment:
`path.extname()` return the file extension beginning with a dot, so you'll have to prefix all of the extensions with a dot. This makes me wonder: Have you not tried running this yet?